renovate.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Renovate configuration for keras image similarity training repository",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":rebaseStalePrs"
+  ],
+  "schedule": [
+    "before 3am every weekday"
+  ],
+  "enabledManagers": [
+    "github-actions",
+    "pip_requirements",
+    "dockerfile"
+  ],
+  "packageRules": [
+    {
+      "matchManagers": ["dockerfile"],
+      "groupName": "Docker"
+    },
+    {
+      "matchManagers": ["github-actions"],
+      "groupName": "GitHub Actions"
+    },
+    {
+      "matchPackageNames": ["tensorflow"],
+      "groupName": "Tensorflow"
+    },
+    {
+      "matchFiles":["training_siamese_network.ipynb"],
+      "matchManagers": ["pip-requirements"],
+      "groupName": "Python for Jupyter Notebook"
+    }
```

Comment:
Second code change required based on my above comment. Thanks for putting this together! 
```suggestion
    }
```